src/Components/Onboarding/ItemLink.tsx

```diff
@@ -77,21 +77,34 @@ export default class ItemLink extends React.Component<Props, State> {
         <Col>
           {
             <Avatar
-              src={this.props.image_url ? this.props.image_url : "https://www.artsy.net/images/icon-70.png"}
+              src={
+                this.props.image_url
+                  ? this.props.image_url
+                  : "https://www.artsy.net/images/icon-70.png"
+              }
               width={50}
               height={50}
             />
           }
         </Col>
         <FullWidthCol>{this.props.name}</FullWidthCol>
         <Col>
-          {
-            this.state.selected ?
-              <Icon name="follow-circle.is-following" color="black" fontSize="39px" /> :
-              <CircleIconContainer>
-                <CircleIcon name="close" color="black" fontSize="21px" style={{ transform: 'rotate(45deg)' }} />
-              </CircleIconContainer>
-          }
+          {this.state.selected ? (
+            <Icon
+              name="follow-circle.is-following"
+              color="black"
+              fontSize="39px"
+            />
+          ) : (
+            <CircleIconContainer>
+              <CircleIcon
+                name="close"
+                color="black"
+                fontSize="21px"
+                style={{ transform: "rotate(45deg)" }}
```

Comment:
It looks like this was just a change from Prettier but I'll note that we could style this with `styled-components` since we're using them heavily in this project. It could look something like -- 
```
const StyledCircleIcon = styled(CircleIcon)`
  transform: rotate(45deg);
`
```
provided that `CircleIcon` passes in `className={this.props.className}` into the top level div. Not a blocker!

Reply:
Yeah I like that. I'm pretty sure @katarinabatina will have some QA items related to these icons so we could probably bundle that refactor in with those items.